xchange-stream-poloniex2/src/main/java/info/bitrich/xchangestream/poloniex2/PoloniexStreamingService.java

```diff
@@ -75,6 +75,27 @@ public boolean processArrayMassageSeparately() {
     return false;
   }
 
+  @Override
+  public void messageHandler(String message) {
+    LOG.debug("Received message: {}", message);
+    JsonNode jsonNode;
+
+    // Parse incoming message to JSON
+    try {
+      jsonNode = objectMapper.readTree(message);
+    } catch (IOException e) {
+      LOG.error("Error parsing incoming message to JSON: {}", message);
+      return;
+    }
+
+    if (jsonNode.isArray() && jsonNode.size() < 3) {
```

Comment:
for these two equality comparisons can you switch the constant to the left side for NPE protetction?

Reply:
fixed